spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/MappingSpannerReadConverter.java

```diff
@@ -47,46 +46,69 @@
 class MappingSpannerReadConverter extends AbstractSpannerCustomConverter
 		implements SpannerEntityReader {
 
+	// @formatter:off
```

Comment:
I totally didn't know that was a thing! i had been manually pressing return on this line the whole time after the formatter messes it up.

Reply:
I know - I got soooo annoyed with it too :)